ui/package.json

```diff
@@ -47,6 +47,7 @@
     "@ember/render-modifiers": "^1.0.2",
     "@glimmer/component": "^1.0.2",
     "@glimmer/tracking": "^1.0.2",
+    "@hashicorp/ember-flight-icons": "^0.0.2-beta",
```

Comment:
update: we're at a 1.0.0 now and this could be updated!

but will wait to make further changes til y'all are working on it (no rush on our end)